haiku-misc/gadgeteer/gadgeteer-0.8.1.recipe

```diff
@@ -0,0 +1,56 @@
+SUMMARY="Command line for creating GUI forms"
+DESCRIPTION="Gadgeteer is a command line app for creating small GUI forms \
+that users can fill out or play with. Once the users press the \"DONE\" \
+button at the bottom of the screen, the command returns the user's input \
+values so they could be used for different purposes."
+HOMEPAGE="https://www.goodeveca.net/beos/"
+COPYRIGHT="2016 Pete Goodeve
+	2006 Kirill Simonov"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://www.goodeveca.net/beos/Gadgeteer_${portVersion}_SRC.zip"
+CHECKSUM_SHA256="9e7b200b157dba04946c786d9205aed3c0dfff50936fa01b2cbaf71683187965"
+SOURCE_DIR="SRC"
+PATCHES="gadgeteer-$portVersion.patchset"
+ADDITIONAL_FILES="gadgeteer.rdef.in"
+
+ARCHITECTURES="x86_gcc2 !x86_64"
+
+PROVIDES="
+	gadgeteer = $portVersion
+	cmd:gadgeteer = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:cc
+	cmd:make
+	"
+
+BUILD()
+{
+	local MAJOR="`echo "$portVersion" | cut -d. -f1`"
+	local MIDDLE="`echo "$portVersion" | cut -d. -f2`"
+	local MINOR="`echo "$portVersion" | cut -d. -f3`"
+
+	sed \
+		-e "s|@MAJOR@|$MAJOR|" \
+		-e "s|@MIDDLE@|$MIDDLE|" \
+		-e "s|@MINOR@|$MINOR|" \
+		"$portDir"/additional-files/gadgeteer.rdef.in \
+			> gadgeteer.rdef
```

Comment:
Sorry, the indentation in (original) comment https://github.com/haikuports/haikuports/pull/3399#discussion_r235624396 was not correct. (It's fixed now.)
```diff
 		"$portDir"/additional-files/gadgeteer.rdef.in \
-			> gadgeteer.rdef 
+		> gadgeteer.rdef
```
(Can you remove one tab in line 47?)
Thanks!